docs/4.x/tests-github.md

```diff
@@ -0,0 +1,50 @@
+---
+category: Develop
+previous: tests-js-tracker
+next: tests-troubleshooting
+---
+# Running tests/builds on GitHub Action
+
+[Github Action](https://github.com/features/actions) makes it easy to automate all your software workflows, now with world-class CI/CD. Build, test, and deploy your code right from GitHub. Make code reviews, branch management, and issue triaging work the way you want.
+
+## Matomo on GitHub Action
+
+Matomo uses GitHub Action to automatically run its build by various triggers.
+- [Build Tracker JS] Trigger by the comment `build js` into the pull request. That will compress js/piwik.js into matomo.js and piwik.js
+- [Build VUE] Auto trigger, it will build VUE file and commit back to the PR
+- [Composer Update] Auto trigger by cron，it will run `composer update` and commit back to the repo
+- [PHPCS check] Auto trigger, checking PHPCS code quality if not valid will return error details.
+- [Inactive PR] after 14 days, inactive PR will mark as state
+
+
+Each developer is responsible to keep the commits/pull requests build green.
+
+## GitHub Action Scripts
+
+All the GitHub Action build file are locate in `.github/workflows/*.yml`
+
+Each script is triggered by its own conditions, for more details see [Github Action Docs](https://docs.github.com/en/actions)
+
+## Create a new CI script
+
```

Comment:
we also need to mention that any new script should be documented in the list above in `Matomo on GitHub Action`